x-pack/plugins/maps/public/classes/fields/es_agg_field.ts

```diff
@@ -87,7 +87,7 @@ export class ESAggField implements IESAggField {
     return this._esDocField ? this._esDocField.getName() : '';
   }
 
-  async createTooltipProperty(value: string | undefined): Promise<ITooltipProperty> {
+  async createTooltipProperty(value: string | string[] | undefined): Promise<ITooltipProperty> {
```

Comment:
optional nit: Is it possible to create a common parameter type for `value` which could be used across the `createTooltipProperty` functions? Just noticing it's changing across all instances of this function and those same instances have the same return type. Just wondering if we could do the same with common type for the param.

Reply:
Good feedback @dhurley14 -- thanks! I ended up leaving as is since this is within the maps plugin and I didn't want to be changing too much since this has to be backported two versions.